master/pkg/schemas/expconf/experiment_config.go

```diff
@@ -210,8 +210,10 @@ type ResourcesConfigV0 struct {
 	// Slots is used by commands while trials use SlotsPerTrial.
 	RawSlots *int `json:"slots,omitempty"`
 
+	RawIsSingleNode   *bool    `json:"is_single_node"`
 	RawMaxSlots       *int     `json:"max_slots"`
 	RawSlotsPerTrial  *int     `json:"slots_per_trial"`
+	RawSlotsPerTask   *int     `json:"slots_per_task"`
```

Comment:
I wonder if we could reuse `slots` field instead? 
I often use `slots_per_trial` by mistake when I really needed `slots` and vice versa, causing issues.
it does not seem like `slots` + `is_single_node=true` will behave similarly to slots today
`slots` + `is_single_node=false` will be more like `slots_per_task`. thoughts?

Reply:
just using slots makes sense to me